FreeRTOS/Demo/RISC-V-ISP/isp/isp_main.c

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
We can delete this, since we're currently only using the demo dir to build freertos into a library